src/main/java/eu/darkbot/api/objects/Circle.java

```diff
@@ -0,0 +1,45 @@
+package eu.darkbot.api.objects;
+
+import eu.darkbot.api.entities.utils.Area;
+
+public interface Circle extends Area, Locatable{
+
+    double getRadius();
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    default boolean isEmpty() {
+        return getRadius() <= 0;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    default boolean containsPoint(double x, double y) {
+        x -= getX();
+        y -= getY();
+        return Math.sqrt(x * x + y * y) <= getRadius();
```

Comment:
dis is slow, u can pre-calculate radius squared: `radiusSq = radius * radius;` and avoid the sqrt